aws-assume-role

```diff
@@ -7,6 +7,9 @@ MFA_DEVICE=$(cat "${MFA_DEVICE_FILE}")
 AWS_ASSUME_ROLE_SESSION_TIMEOUT=${AWS_ASSUME_ROLE_SESSION_TIMEOUT:-7200}
 AWS_ASSUME_ROLE_DEFAULT_REGION=${AWS_ASSUME_ROLE_DEFAULT_REGION:-us-west-2}
 
+AWS_ASSUME_ROLE_CACHE_TO_FILE=${AWS_ASSUME_ROLE_CACHE_TO_FILE:-false}
+AWS_ASSUME_ROLE_CACHE_FILE=${AWS_ASSUME_ROLE_CACHE_FILE:-~/.aws/.aws-assume-role-master-creds}
```

Comment:
I think it should default to a file in /tmp to get wiped across reboots

Reply:
How would we know what the file is across terminal sessions?

Reply:
It can be a predefined name, I'm just saying we should put it in /tmp instead of the home directory.

Reply:
ah, gotcha